scipy/sparse/linalg/_isolve/lsqr.py

```diff
@@ -281,7 +281,7 @@ def lsqr(A, b, damp=0.0, atol=1e-6, btol=1e-6, conlim=1e8,
     array([ 0.,  0.])
 
     The stopping code `istop=0` returned indicates that a vector of zeros was
-    found as a solution. The returned solution `x` indeed contains `[0., 0.]`.
+    found as a solution. The returned solution `x` indeed contains ``[0., 0.]``.
     The next example has a non-trivial solution:
```

Comment:
This line is above 80
```suggestion
    found as a solution. The returned solution `x` indeed contains
    ``[0., 0.]``. The next example has a non-trivial solution:
```